<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeSet</class>
 <widget class="QDialog" name="TimeSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1796</width>
    <height>841</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>30</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>741</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>420</y>
     <width>481</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="timeSpec">
    <enum>Qt::TimeZone</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>140</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>锁屏开关：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1260</x>
     <y>150</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/lockcreenOFF.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>270</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>31</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>(CST+800)北京</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>(JSC +900)东京</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>(EDT- 400)纽约</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>(CDT-500)芝加哥</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>(MDT-600)丹佛</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>(COT-500)波哥大</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>(CDT-400)哈瓦那</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>(EST-500)巴拿马</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>(EST+200)巴黎</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>(EST+200)布达佩斯</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>(CEST+200)柏林</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>(MSK+300)莫斯科</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>290</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>选择时区：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>430</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>31</pointsize>
    </font>
   </property>
   <property name="text">
    <string>设置时间：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1380</x>
     <y>700</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/btexit.png);</string>
   </property>
   <property name="text">
    <string>   返回</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>700</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/confirmbt.png);
</string>
   </property>
   <property name="text">
    <string>  确定</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
